docs/radar-protocols/navico/protocol.md

```diff
@@ -119,6 +119,38 @@ Doppler modes:
 | 1 | Both (show approaching and receding) |
 | 2 | Approaching only |
 
+## Control Categories
+
+Radar settings are categorized by their purpose and persistence:
+
+### Installation Settings (Report 04)
+These are configured once during radar installation and rarely changed:
+- **Bearing alignment** - Corrects for antenna mounting offset (deci-degrees, 0-3599)
+- **Antenna height** - Height above waterline in millimeters (affects sea clutter calculations)
+- **Accent light** - HALO pedestal LED brightness (0-3, HALO only)
+
+### Runtime Controls (Report 02)
+Operational settings adjusted during normal use:
+- **Gain** - Signal amplification (0-100%, auto or manual)
+- **Sea clutter** - Sea return suppression (auto: harbor/offshore, or manual 0-100%)
+- **Rain clutter** - Precipitation suppression (0-100%)
+- **Interference rejection** - Filter other radar interference (off/low/medium/high)
+- **Target expansion** - Make small targets more visible (off/on)
+- **Target boost** - Amplify weak targets (off/low/high)
+
+### Advanced Settings (Report 08)
+Performance tuning options:
+- **Sidelobe suppression** - Reduce sidelobe artifacts (0-100%, auto or manual)
+- **Scan speed** - Antenna rotation speed (normal/fast)
```

Comment:
For HALO the scan speed is an enum with 4 values:
0 = Normal, 1 = Medium, 2 = Medium High, 3 = High 
4G has 3 values, older models probably even fewer:
0 = Normal, 1 = Medium, 2 = Medium High